src/applications/static-pages/widgetTypes.js

```diff
@@ -6,7 +6,6 @@ export default {
   CORONAVIRUS_CHATBOT: 'va-coronavirus-chatbot',
   COVID_VACCINE_UPDATES_CTA: 'va-covid-vaccine-updates-cta',
   CTA: 'cta',
-  DIRECT_DEPOSIT: 'direct-deposit',
```

Comment:
I confirmed that this react widget is no longer used on https://www.va.gov/change-direct-deposit/